flash/data/data_module.py

```diff
@@ -215,7 +215,8 @@ def _train_dataloader(self) -> DataLoader:
         return DataLoader(
             train_ds,
             batch_size=self.batch_size,
-            shuffle=True,
+            shuffle=False if isinstance(train_ds, (IterableDataset,
```

Comment:
Can just do `shuffle = not isinstance(train_ds, (IterableDataset, IterableAutoDataset))`?

Reply:
Done !